src/plots/rectanglePlot.ts

```diff
@@ -251,6 +253,50 @@ export module Plots {
       });
     }
 
+    /**
+     * Gets the Accessor for label.
+     */
+    public label(): AccessorScaleBinding<any, string>;
```

Comment:
We probably need to revisit the property API for a later v2.0.

Let's consider what a label is supposed to do and what properties are supposed to do and see if the two have aligned goals.

My interpretation is that properties (using `x()` and `y()` as examples) derive aspects from the data and then can be driven into a scale so that the data is displayed properly.

Labels, however, derive aspects from the data and should be perfectly fine without the need of a scale.

Thus, label should not be a property.

Hopefully that makes sense.